cornflow-server/cornflow/app.py

```diff
@@ -72,9 +72,9 @@ def _fk_pragma_on_connect(dbapi_con, con_record):
     for res in resources:
         api.add_resource(res["resource"], res["urls"], endpoint=res["endpoint"])
 
-    docs = FlaskApiSpec(app)
-    for res in resources:
-        docs.register(target=res["resource"], endpoint=res["endpoint"])
+    # docs = FlaskApiSpec(app)
+    # for res in resources:
+    #     docs.register(target=res["resource"], endpoint=res["endpoint"])
```

Comment:
This is the only that should be reviewed. I have not been able to run it with flask api swagger active.